terraform/environments/oasys/locals_secrets.tf

```diff
@@ -38,6 +38,11 @@ locals {
     resources = ["*"]
   }
 
+  secretsmanager_secrets_bip = {
```

Comment:
i think it'd make sense this just being 
secretsmanager_secrets_default = {
  secrets = {
    passwords = {}
  }
}